tests/e2e/ios-objc/Podfile

```diff
@@ -1,13 +1,11 @@
-use_frameworks!
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
 
-target 'TypewriterExample' do
-  pod "Analytics"
-  platform :ios, "10.1"
-end
+target 'RudderTyperExample' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
 
-target 'TypewriterExampleTests' do
-  pod "Analytics"
-  platform :ios, "10.1"
+  # Pods for RudderTyperExample
+  pod 'Rudder', '1.28.0'
```

Comment:
Don’t fix the Rudder-iOS SDK version; make it flexible up to `2.0.0`.
In case you want to fix the version then please document the steps to update the version in readme.